core_backend/tests/api/test_question_answer.py

```diff
@@ -542,6 +541,7 @@ class TestSTTResponse:
         [
             ("correct", 200, {"text": "Paris"}),
             ("incorrect", 401, {"error": "Unauthorized"}),
+            ("correct", 400, {"text": "Paris"}),
```

Comment:
I think calling whether the api is correct or not `outcome` is confusing. Maybe you can make it a boolean